frontend/src/main/components/PersonalSchedule/PersonalScheduleForm.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import { Button, Form } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import { useNavigate } from "react-router-dom";
+import SingleQuarterDropdown from "../Quarters/SingleQuarterDropdown";
+import { quarterRange } from "../../utils/quarterUtilities";
+
+function PersonalScheduleForm({
+  initialPersonalSchedule,
+  submitAction,
+  currentUser,
+  buttonLabel = "Create",
+}) {
+  // Stryker disable all
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+    setValue,
+  } = useForm({ defaultValues: initialPersonalSchedule || {} });
+  // Stryker enable all
+
+  const navigate = useNavigate();
+  const [quarter, setQuarter] = useState(
+    {
+      quarters: quarterRange("20221", "20224"),
+    }.quarters[0]
+  );
```

Comment:
I wonder if the default value here probably also needs to match the default value in the SingleQuarterDropdown just like in the other PR (otherwise I wonder if we'll have the same bug?)

Reply:
fixed